ert3/storage/_storage.py

```diff
@@ -3,29 +3,98 @@
 from pathlib import Path
 from typing import Any, Dict, Iterable, Optional, cast, Set
 import io
-
+import logging
 import cloudpickle
 import requests
 import ert3
 
+from ert_shared.storage.connection import get_info
+
+logger = logging.getLogger(__file__)
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
```

Usually I see `__name__` which is the module name and not file path used in logging.